docs/gadgets/ebpftop.md

```diff
@@ -0,0 +1,50 @@
+---
+# Code generated by 'make generate-documentation'. DO NOT EDIT.
+title: Gadget ebpftop
+---
+
+ebpftop shows cpu time used by ebpf programs.
+
+The following parameters are supported:
+ - interval: Output interval, in seconds. (default 1)
+ - max_rows: Maximum rows to print. (default 20)
+ - sort_by: The field to sort the results by (all,runtime,runcount,programid). (default all)
```

Comment:
Could you use those parameters in `pkg/resources/samples/trace-ebpftop.yaml` to show how they can be used?

There is an example in another gadget: pkg/resources/samples/trace-socket-collector.yaml

I think numbers in the yaml have to be in quotes since `Parameters` is a `map[string]string`.

Reply:
Yeah, I will add that. Just saw that I was missing a block of code that is using these values in the gadget itself (and not only in the kubectl-gadget). Will add that as well :)